sw/device/lib/testing/mock_mmio.h

```diff
@@ -141,6 +141,20 @@ Int ToInt(std::initializer_list<BitField> fields) {
  */
 inline internal::LittleEndianBytes LeInt(const char *bytes) { return {bytes}; }
 
+Int Inverted(const char *bytes) {
+  char *inverted_bytes = malloc(sizeof(Int));
+  Int ret_val;
+
+  for (int i = 0; i < sizeof(Int); i++) {
+    inverted_bytes[i] = ~bytes[i];
```

Comment:
Perhaps it might be easier to do the memcpy first and then ~ the entire integer? I.e.
```cc
Int val;
memcpy(&val, bytes, sizeof(val));
return ~val;
```